### ISSUE TITLE
GitcoinGrant18 Giveth comms 

### ISSUE THREAD

**Issue by oyealmond** [Giveth/giveth-planning#1511]
Hey @SnakeGH! 
@laurenluz is working on Giveth proposal for GitcoinRound18, and wanted to coordinate w the fundraising team the comms efforts happening this time. Usually we have tweets, jump in Twitter Spaces, and GIveth Galaxy Participating projects promotion (regularly w a Medium post). 

Let me know if there's smth special needed for this round. Thanks

**Comment by laurenluz** [comment]
The round is from August 15-29 and 300,000 DAI is available in the OSS round. I just applied for giveth - here is all the info I used: https://docs.google.com/document/d/1awsOwcKDcpTggPLRP1ggjdoeDMiMXv7PQQCjuiGvGXs/edit

I also applied for the Zuzalu round which has 25,000 DAI in it and I might apply as well for LatAm - also 25,000 DAI.

I can hopefully let you know where we get accepted. 

We should make some tweets, for exaple:
- a summary of our achievements from last quarter & plans for the future (check the google doc for desriptions of this, I wrote it) - this would be good to get out on August 15 when the round goes live and pin it to our twitter @SteffanThorn @oyealmond 
- twitter thread cart recommendations for giveth galaxy projects (we need to see which are in which round)
- basic reminders about the round being open & when it will close (e.g. just 1 more day to donate, here's our grant link!)

We should also:
- make an annoucement or two in the discord
- mention in the community call to tell people to donate & when
- promote on instagram & linkedIn
- get giveth representatives in as many "shill your grant" twitter spaces as possible (we should make a doc to track them and get volunteers)

**Comment by SnakeGH** [comment]
GM @oyealmond sounds like Lauren was able to answer all your thoughts / questions for Giveth. 

Assuming you wouldn't handle any comms for the other DAOs in our fundraising call team, so do you need anything else from the specific Giveth fundraising team?

**Comment by SteffanThorn** [comment]
I've provided 2 ways we could present the achievements and future works. 

1st, long tweet. Basically a copy-paste from the doc @laurenluz. We'll need to manually post the tweets. One after the other as twitter doesn't allow threads with long tweets, but you can comment a long tweet under a long tweet (loophole). I can facilitate this in my evening. 

2nd, 2 thread -- 1 thread with past achievements, 1 thread with future works. Gives us a little more detail we can add in each part (there are 2 parts I don't know what to add and I don't fully understand). The 2nd thread can be scheduled after the 1st thread is posted so we can include the 1st thread in the 1st tweet of the 2nd thread -- like a quote retweet. 

Also. I've noticed people using the hashtag #GG18 for this event, hence why I've included it.

https://docs.google.com/document/d/1LOvFCHLDI0MTGuckGXmPOS1epLGY3GxbbOf4S1voTMQ/edit

**Comment by SteffanThorn** [comment]
@laurenluz how do we find out about the cart rounds?

**Comment by oyealmond** [comment]
Hey @laurenluz! Thank you so much for your guidance. 
Nothing else needed @SnakeGH 

Thanks for drafting tweets @SteffanThorn! Im reviewing them rn.

**Comment by laurenluz** [comment]
I made edits on all! I think the 1st thread (either Almond's or Stee's after edits) are good to go.

For the 2nd one, with all the extra details, I think it's great to talk about the "in progress goals" but then for the "future ideas", we should cut it down a lot... or maybe make it like a one or two tweets with bullets saying "we're also investigating: NFT donations, fiat, reputation systems, etc.)... instead of having it so expanded.

Basically because those bullets are just IDEAs. making it into a long thread like that I feel is us committing too much 😅 . We have to be careful to always underpromise, overdeliver.

Anyway, I made edits to those tweets as well.

**Comment by laurenluz** [comment]
@SnakeGH it would be great if you could keep an eye on which galaxy projects are in the round(s) and then relay that info to @SteffanThorn & @oyealmond so they can make a "support galaxy projects" thread.

**Comment by SnakeGH** [comment]
@laurenluz will ask the team about it on Monday and report back to them :)

**Comment by oyealmond** [comment]
thanks @SteffanThorn I'll be posting this tmrrw, and will follow up.

**Comment by SnakeGH** [comment]
> @SnakeGH it would be great if you could keep an eye on which galaxy projects are in the round(s) and then relay that info to @SteffanThorn & @oyealmond so they can make a "support galaxy projects" thread.

On yesterdays call, which was very light on attendees, these are the current projects under the Giveth Galaxy umbrella that we know have applied for the Gitcoin round:

**Giveth, Pairwise, Fair Data Society, Gravity, ETHColombia, LunCo. & AstroCool** have applied

Ill follow up again next week when some more core contributors are back from vacay

**Comment by SnakeGH** [comment]
@laurenluz Im having a hard time locating Giveth's Gitcoin donation location on their website, do you have a quick link to where it is located?

**Comment by oyealmond** [comment]
Links are working now @SnakeGH @laurenluz 
👇🏼
https://explorer.gitcoin.co/#/round/10/0x8de918f0163b2021839a8d84954dd7e8e151326d/0x8de918f0163b2021839a8d84954dd7e8e151326d-86

**Comment by oyealmond** [comment]
both @SteffanThorn threads are posted. 

@danelsuga can you review[ this thread ](https://docs.google.com/document/d/1pyiyfiIcRTAq2oBftQ3dmID_HCa1Y-JuS9Dol7iVKK4/edit?usp=sharing)on OSS GG18 projects?
